danceradvent/public/articles/2023/21-dancer-core-team-partnership-with-tprf.pod

```diff
@@ -0,0 +1,70 @@
+=head1 Tidings of Comfort and Joy: Dancer Core Team, TPRF Partnership
+
+In the lead up to Dancer2 1.0.0, the Dancer Core Team finalized an
+arrangement with The Perl and Raku Foundation (TPRF) to have the
+foundation collect and manage funds on behalf of the Dancer project.
+
+=head2 Background
+
+We're a small team, and setting up and managing a non-profit corporation
+is a significant time investment. Every minute we spend managing funds,
+paying bills, etc. is time we aren't spending advancing Dancer. TPRF
+has all of this infrastructure in place already, and such a partnership
+helps TPRF fulfill its goal of supporting the Perl and Raku communities.
```

Comment:
"... such a partnership" - I would want to hear more about this partnership before discussing it in this way. Maybe a sentence here that we are partnering to allow TPRF to handle this administrative work we just mentioned?